# en_US resources
# the signature must not be deleted
signature="sIgNaTuRe"
abort = "Execution aborted"
base.error = "Base error"
cert.invalid = "Syntax error in %1"
cert.pem = "Failed to open %1 file: %2"
cert.read = "Failed to read the certificate from %1"
cert.verification = "Signature does not validate against certificate (signing is compromised)"
cert.write = "Failed to write certificate"
disklib = "Unable to initialize DiskLib"
eula.accept = "Failed to accept EULA"
eula.read = "EULAs not read and therefore not accepted"
http.curl = "cURL error: %1"
http.method = "Unsupported HTTP method"
http.open = "Could not open URL %1 (response code: %2)"
http.post = "Bad response code (%1) from POST request"
http.unexpected.end.of.stream = "Network connection was lost."
http.send = "Lost connection to target"
http.receive = "Lost connection to source"
internal.configOption = "Internal error: Failed to open config option file at %1"
internal.connection = "Internal error: Failed to connect to server"
internal.fault.notfound = "Internal error: FaultType not found: %1"
internal.hardware = "Internal error: Hardware version not found"
internal.importspec = "Internal error: Unexpected import spec"
internal.initlibs = "Internal error: Unable to initialize libs. Reason: %1"
internal.install = "Internal error: Failed to open file %1. Corrupted installation?"
internal.maxRead = "Internal error: Contents too long"
internal.mem = "Internal error: Out of memory"
internal.object = "Internal error: Unknown object"
internal.ova.padding.space = "Internal error: Not enough room to pad descriptor"
internal.searchobject = "Internal error: Failed to create search object for %1"
internal.stream = "Internal error: Unexpected end of stream"
internal.uncatch = "Internal error: Uncaught exception: %1"
internal.unknownobject = "Internal error: Unknown object"
internal.zlib = "Internal error: Unable to initialize zlib"
internal.digest = "Internal error: Cannot update sha digest as final digest have been made."
internal.resume.file.not.found = "Internal error: Resuming download, but target %1 was not found."
io.dir.create = "Failed to create directory: %1"
io.dir.exist.target = "Target must be an existing directory"
io.disk.adapter.type = "Invalid target disk adapter type: %1"
io.disk.capacity = "Capacity mismatch for disk %1"
io.disk.create = "Failed to create disk: %1. Reason: %2"
io.disk.db = "Failed to read DB from disk: %1. Reason: %2"
io.disk.grainsize = "Invalid grain size for %1"
io.disk.info = "Failed to get disk info from disk: %1. Reason: %2"
io.disk.not.found = "Disk (%1) could not be found."
io.disk.open = "Failed to open disk: %1"
io.disk.open.reason = "Failed to open disk: %1. Reason: %2"
io.disk.read = "Failed to stream from disk: %1. Reason: %2"
io.disk.write = "Failed to write content to disk %1. Reason: %2"
io.file.chunkSize = "Invalid file length for %1 (max size is: %2 bytes)"
io.file.compress = "Failed to compress file"
io.file.decompress = "Failed to decompress file"
io.file.exist = "File already exists: %1"
io.file.exist.output = "Output file %1 already exists. Use overwrite flag to delete it."
io.file.not.found = "File (%1) could not be found."
io.file.open = "Failed to open file: %1"
io.file.read = "Failed to read from file: %1"
io.file.rename = "Failed to rename file %1 to %2"
io.file.write = "Failed to write content to file %1. Reason: %2"
io.file.permission = "Access denied for: '%1'"
locator.forward = "URL forwarded to resource with unknown protocol: %1"
locator.host = "Server not specified: %1"
locator.object = "Locator does not refer to an object: %1"
locator.parse = "Cannot parse locator: %1"
locator.port = "Invalid port number: %1"
locator.protocol = "Unsupported protocol: %1"
locator.query = "Unknown locator query key: %1"
locator.query.key.missing = "Missing query key in locator: %1"
locator.scheme = "Invalid scheme for %1 locator: %2"
logLevel = "Invalid log level specified: %1"
login = "Failed to login: %1"
login.abort = "Login aborted"
login.attempt = "Too many login attempts"
login.machine.user = "Missing username in locator %1"
login_vcloud.ticket.version = "Server '%1' does not support login by ticket"
manifest.line = "Invalid manifest file (line: %1)"
manifest.mismatch = "SHA1 digest of file %1 does not match manifest"
media.media.source.to.ovf.target = "Uploading a media source to an OVF destination is not supported."
media.ovf.source.to.media.target = "Uploading an OVF source to a media destination is not supported."
media.probe = "Probing media files is not supported"
media.iso.target.file.only = "Only local files are allowed as media targets."
media.source.target.compat = "Can not upload %1 source to %2 target. Both need to be either ISO or FLP."
myvmw.api.version.not.supported = "Unsupported my.vmware.com service: %1"
myvmw.communication.response = "Unexpected response code from my.vmware.com: %1"
myvmw.communication.error = "Unable to communicate with my.vmware.com. Error code: %1, message: %2"
myvmw.connect.version = "Failed to retrieve my.vmware.com service version."
myvmw.href.failed = "Failed to get OVF package href from my.vmware.com response."
myvmw.invalid.response = "Unexpected response message from my.vmware.com: %1"
myvmw.not.entitled.for.download = "The provided user is not entitled to download the product. Message: %1"
myvmw.no.compat.product.found = "Unable to find compatible product for input product. Message: %1"
net.lookup = "Could not lookup host: %1"
net.connect = "Failed to connect to host %1 on port %2."
nfc.agent = "Invalid service agent type"
nfc.connection = "Unable to establish NFC/HTTP-NFC connection"
nfc.source.db = "Unable to read disk database from NFC/HTTP-NFC source"
nfc.source.read = "Error reading from NFC/HTTP-NFC source"
nfc.target.db = "Unable to set disk database from NFC/HTTP-NFC target"
nfc.target.disk = "Unable to open NFC/HTTP-NFC target disk"
nfc.target.write = "Error writing to NFC/HTTP-NFC target"
nfc.ticket.target = "Unable to get NFC/HTTP-NFC ticket for target disk"
op.disk.db.unknown = "Failed to read DB from disk: %1"
option.capacity = "Invalid capacity value '%1'. Should be one of b, kb, mb, gb, tb"
option.capacity.big = "Capacity too big"
option.capacity.zero = "Capacity is zero"
option.extraconfig = "%1 with value %2 is not a supported ExtraConfig for target"
option.fileRef = "File referenced by %1 does not exist: %2"
option.format = "Failed to parse option (%1). Format of value (%2) could not be parsed."
option.overwrite = "The overwrite (-o) option is not supported with this target"
option.parse.bool = "Value not a Boolean"
option.parse.config = "Failed to parse option '%1' in configuration file '%2'"
option.parse.illegal = "Value is illegal for option %1. Possible values are: %2"
option.parse.invalid = "Invalid value for %1 option"
option.parse.negInt = "Value must not be negative: %1"
option.parse.posInt = "Value must be greater than zero: %1"
option.parse.reason = "Failed to parse option '%1'. Reason: %2"
option.parse.source = "Source type not known: %1"
option.parse.target = "Destination type not known: %1"
option.parse.uri = "Value not a valid URI"
option.parse.uri.auth = "URI contains an invalid authority part: %1"
option.parse.uri.host = "No host specified in the URI"
option.source.target = "No source or target specified. Try 'ovftool --help' for more options."
option.unexpected = "Unexpected option: %1"
option.unknown = "Unknown option: '%1'"
ova.descriptor.open = "Did not find an .ovf file at the beginning of the OVA package."
ovf.delta.capacity = "Capacity of disk differs from specified value in OVF descriptor: %1"
ovf.delta.create = "Cannot create delta disks: Incompatible disk grain sizes"
ovf.delta.grain = "Unable to allocate grain buffer"
ovf.delta.ova = "Delta disk creation does not work with an OVA source"
ovf.delta.ref = "Multiple disk elements have the same file references"
ovf.delta.source = "Source already has delta disk compression"
ovf.descriptor.open = "Failed to open OVF descriptor"
ovf.file.device = "Could not find file for device id: %1"
ovf.local.ref = "OVF has a local file reference"
ovf.network.notfound = "Network name not defined in OVF descriptor: %1"
ovf.parse = "Errors found while parsing OVF descriptor"
ovf.stringbundle.open = "Failed to open string bundle: %1"
ovf.target = "OVF Package is not supported by target"
ovf.version = "The OVF destination requires that the source OVF is in the version 1.0 format"
system.error = "System error: (%1)"
tar.checksum = "Invalid header block: %1 != %2"
tar.create = "Tar entry not created"
tar.eof = "End of file reached while skipping entry"
tar.name = "Cannot create header block. Name too long: %1"
tar.size = "Cannot handle OVA archive. Size of %1 is too big"
target.directory = "Target directory does not exist: %1"
target.exist = "Target already exists: %1"
target.name = "Unable to determine name for target"
transform.extraconfig.allow = "ExtraConfig options exists in source (use --%1 option to allow this)."
vapprun.empty.name = "Unable to determine name of target"
vapprun.exist = "VM or vApp with name '%1' already exists"
vapprun.locator = "Not a valid vapprun workspace locator: %1"
vapprun.networks = "The source contains more than one network. This target supports at most one network."
vapprun.notfound = "Did not find entity: %1"
vapprun.workspace.invalid = "Not a valid vapprun workspace: %1"
vapprun.workspace.version = "The vapprun workspace version is not supported"
vcloud.catalog.media = "Media name '%1' not found in the catalog."
vcloud.catalog.media.exist = "Media name already found in the catalog. Either overwrite it or choose a different name."
vcloud.catalog.multiple = "Multiple catalogs found. Possible catalog completions are: %1"
vcloud.catalog.notfound = "Catalog not found: %1"
vcloud.catalog.vapp = "vApp name '%1' not found in the catalog"
vcloud.catalog.vapp.exist = "vApp name already found in the catalog. Either overwrite it or choose a different name."
vcloud.catalog.overwrite.not.vapp = "Can not overwrite a catalog item, which does not contain a vApp template entity."
vcloud.catalog.overwrite.not.media = "Can not overwrite a catalog item, which does not contain a media entity."
vcloud.invalid.response = "Invalid response from server"
vcloud.invalid.response.message = "Invalid response from server: %1"
vcloud.invalid.vendor = "Invalid response from server. Vendor specific error code: %1"
vcloud.locator.key = "Missing vappTemplate or vapp key in locator"
vcloud.locator.key.media = "Missing media key in locator"
vcloud.locator.key.any = "Missing media, vappTemplate, or vapp key in locator"
vcloud.locator.media.vapp = "The use of the media key in the locator is not allowed with keys: vapp and vappTemplate"
vcloud.network.notfound = "Network name not found on target: %1"
vcloud.org = "No organization specified: %1"
vcloud.ovf.property.notsupported = "Source contains OVF properties that are not supported by the vCloud target."
vcloud.server.catalog = "No catalog found on the server."
vcloud.server.notsupported = "Source type not supported with target. Convert to OVF/OVA first"
vcloud.server.timeout = "Timed-out waiting for server"
vcloud.server.vapp = "No vApp found on server"
vcloud.server.vapptemplate = "No vAppTemplate found on server"
vcloud.server.vdc = "No VDC found on server"
vcloud.source.sslfingerprint = "Source SSL fingerprint mismatch detected (expected: %1, actual: %2)."
vcloud.source.vapp = "Source vApp is running"
vcloud.source.media = "The source type has be to either ISO or FLP. Source type %1 was given to vCloud media target."
vcloud.source.media.not.supported = "Downloading media files from vCloud Director is only supported on version 6.0 or newer."
vcloud.storage.profile.notfound = "VDC storage profile not found: %1"
vcloud.task.fail = "Task failed (%1:%2) : %3"
vcloud.target.duplicate = "Duplicate name on target. Use overwrite flag to delete it"
vcloud.target.multiple = "Multiple networks found on target. Possible completions are: %1"
vcloud.target.networks = "No networks found on target"
vcloud.target.sslfingerprint = "Target SSL fingerprint mismatch detected (expected: %1, actual: %2)."
vcloud.target.vapp = "Target vApp is running"
vcloud.url.ovf = "No OVF descriptor URL received"
vcloud.user.cannotupload = "The user %1 cannot upload to the vCloud server"
vcloud.user.cannotupload.noname = "The user cannot upload to the vCloud server"
vcloud.vapp.multiple = "Multiple vApps found. Possible vApp completions are: %1"
vcloud.vapp.notfound = "vApp not found: %1"
vcloud.vapptemplate.multiple = "Multiple vAppTemplates found. Possible vAppTemplate completions are: %1"
vcloud.vapptemplate.notfound = "vAppTemplate not found: %1"
vcloud.vdc.invalid = "Invalid VDC response: %1"
vcloud.vdc.multiple = "Multiple VDCs found. Possible VDC completions are: %1"
vcloud.vdc.notfound = "VDC not found: %1"
vcloud.version = "No supported vCloud version was found for locator: %1"
vi.communicate = "Failed to communicate with VI server"
vi.datacenter = "Datacenter object not found"
vi.device.path = "Could not generate path for device %1"
vi.import.vapp = "Import of vApp %1 is not supported by the host"
vi.moref.format = "Invalid moref format for option --%1:%2"
vi.moref.type = "Invalid moref type for option --%1:%2"
vi.namespace.version = "No supported namespace version was found on host. Max supported version: %1."
vi.object = "Found wrong kind of object (%1). Possible completions are: %2"
vi.object.invalid = "Invalid object type specified (not a ManagedEntity subtype)"
vi.pemfile = "Invalid PEM file: %1."
vi.remote.moref = "A connection to the remote object could not be made. OVF Tool does not support this server."
vi.remote.object = "Could not make a connection to the server managed object %1. OVF Tool does not support this server"
vi.source.disk = "Source disk not found"
vi.source.disk.open = "Failed to open file stream: %1"
vi.source.lease = "Got fault from server: %1"
vi.source.operation = "Operation is not supported on source"
vi.source.ovf = "Server failed to return OVF descriptor"
vi.source.ticket = "Invalid source session ticket"
vi.sslfingerprint = "Provided SSL fingerprint (%1) did not match at locator %2 with fingerprint (%3)."
vi.sslfingerprint.delete = "Could not delete SSL fingerprint for host (%1). Remove entry in '%2'."
vi.target.createvm = "Failed to create VM %1. Reason: %2"
vi.target.createvm.invalidcfg = "Failed to create VM %1. Reason: '%2' for property '%3'"
vi.target.datastore.empty = "No datastores found on target"
vi.target.datastore.invalid = "Invalid target datastore specified (%1): Possible completions: %2"
vi.target.datastore.specified = "No target datastore specified. Possible completions: %1"
vi.target.disk = "Target disk not found"
vi.target.diskmode = "Disk mode: %1 not supported by host. Supported disk modes: %2"
vi.target.diskmode.unsupported = "Unsupported disk mode specified"
vi.target.diskmode.version = "Disk mode '%1' is not supported for hardware version %2. Minimum supported hardware version is %3"
vi.target.duplicate.object = "Duplicate name on host with a Folder or Resource Pool object (overwrite flag is ignored)"
vi.target.file = "Target does not support upload of non-disk files. Use --noImageFiles to disable this"
vi.target.host.cluster = "No target host found in cluster that supports the chosen network/datastore/virtual hardware combination. Possible completions: %1"
vi.target.host.empty = "Found no hosts in target cluster to deploy on. Cannot deploy to an empty cluster"
vi.target.host.maintenance = "Found no hosts in target cluster to deploy on. All hosts in the cluster are either disconnected or in maintenance mode"
vi.target.name = "Target name longer than 80 characters: %1. Use option --name to change the target name"
vi.target.network = "Unable to map to target networks. One or more target networks not found. Valid target networks: %1"
vi.target.network.empty = "No target networks. OVF networks: %1. Target networks: %2"
vi.target.network.no.mapping = "No network mapping specified. OVF networks: %1. Target networks: %2"
vi.target.network.ovf = "Invalid OVF name (%1) specified in net mapping. OVF networks: %2. Target networks: %3"
vi.target.network.server = "Invalid target name (%1) specified in net mapping. OVF networks: %2. Target networks: %3"
vi.target.storagepod.empty = "Selected datastore cluster (%1) did not contain any datastores"
vi.target.ticket = "Invalid target session ticket"
vi.target.vmfolder = "Invalid VM folder specified"
vi.task = "Task failed on server: %1"
vi.verifyonly.no.datastore = "No datastore was provided in verifyOnly mode."
vi.vm.invalidcfg = "Invalid VM configuration. Reason: '%1' for property '%2'"
vservice.vsdprovider = "VServiceDependency provider id (%1) not found. List of valid values are: %2"
vservice.vsdprovider.red = "VServiceDependency %1 is not compatible with provider %2: %3"
vservice.vsdsection = "VServiceDependencySection id (%1) not found. List of valid values are: %2"
write = "An error occured while writing to %1. Reason: %2"
xml.invalid = "Line %3 column %4: Invalid value '%1' for attribute '%2'."
xml.missing = "Line %3 column %4: Missing '%1' element in %2."
xml.root = "%1: Unexpected root element"
xml.unexpected = "Line %2 column %3: Unexpected element tag '%1' found."
xml.xerces = "Line %3 column %4: Failed to parse %1. Reason: %4."

